crates/starknet/src/starknet/events.rs

```diff
@@ -420,32 +419,29 @@ mod tests {
     fn dummy_call_info(events_count: usize) -> CallInfo {
         let mut call_info = CallInfo::default();
 
-        for idx in 1..=events_count {
-            let event = starknet_in_rust::execution::OrderedEvent {
-                order: idx as u64,
-                keys: vec![Felt252::from(idx + 10)],
-                data: vec![Felt252::from(idx + 20)],
+        for idx in (1..=events_count).rev() {
```

Comment:
Why do we need to reverse the order?

Reply:
https://github.com/0xSpaceShard/starknet-devnet-rs/commit/9ae41a30df54f9982158722ed593f65f750a3505